### ISSUE TITLE
Language string issue on rebuild form cache button

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic#13706]
### Mautic Version

5.0.x series

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8

### What happened?

There seems to be a bug with the language string on the rebuild cache button for forms:

![screenshot-mautic ddev site-2024 04 29-18_09_51](https://github.com/mautic/mautic/assets/2930593/12be8c64-cbcd-491b-b220-b55df7f1f54b)

### How can we reproduce this issue?

Step 1: Try to rebuild cache for forms (select them, then at the top click to rebuild the cache)

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by kavya-gaur** [comment]
Hi is anyone working on this issue ? I would like to fix the above issue by modifying the language string in the cache button.

**Comment by RCheesley** [comment]
I don't think so @kavya-gaur - please go ahead! Let us know if you need any guidance on getting started, and please use the 5.1 branch for your PR.  Here's some info to get you going: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests.

**Comment by RCheesley** [comment]
Still an issue in 5.2.1, @kavya-gaur did you still want to work on this?

![Image](https://github.com/user-attachments/assets/8bfdf738-63f2-4107-8885-dec365c91695)

**Comment by saschafoerster** [comment]
Solved in Mautic 6.0.0
<img width="428" alt="Image" src="https://github.com/user-attachments/assets/e902307e-b8e5-4774-b500-73e803394043" />
